crux/src/Crux/SVCOMP.hs

```diff
@@ -280,4 +275,4 @@ deduplicateTasks :: [BenchmarkSet] -> TaskMap
 deduplicateTasks = Map.unionsWith (++) . map f
   where
   f bs = Map.fromList
-           [ ((t, benchmarkArchWidth bs), [bs]) | t <- benchmarkTasks bs ]
+           [ (t, [bs]) | t <- benchmarkTasks bs ]
```

Comment:
Should update the comment to this function as well; target architecture is not longer a separate component of the task map.

Reply:
Right you are. Fixed.